ecl/hql/reservedwords.cpp

```diff
@@ -607,9 +608,11 @@ void printKeywordsToXml()
      {
          buffer.append("  <cat group=\"").append(keywordList[i].group).append("\">\n");
          unsigned int j = 0;
+         StringBuffer encodedStr;
          while(keywordList[i].keywords[j])
          {
-             buffer.append("    <keyword name=\"").append(keywordList[i].keywords[j]).append("\"/>\n");
+             encodedStr.clear();
+             buffer.append("    <keyword name=\"").append(encodeXML(keywordList[i].keywords[j], encodedStr)).append("\"/>\n");
```

Comment:
utf8 should probably be set to true, but it isn't going to cause a problem, so will merge as is.